app/static/css/style.css

```diff
@@ -281,3 +281,21 @@ input[type="radio"] {
     vertical-align: middle;
     margin: 0 auto;
 }
+.preview-image{
+    height: 250px;
+    width: 180px;
+    margin-left: 80px;
+    background-size: cover;
+    padding-top: 140px;
```

Comment:
Instead of `padding-top` and `padding left ` we can use `padding: 140px 0 0 5px` 😅

Reply:
doing it